test/selenium-test/sign-in-widget/app.component.html

```diff
@@ -5,8 +5,8 @@
 <link href="https://global.oktacdn.com/okta-signin-widget/4.3.2/css/okta-sign-in.min.css" type="text/css" rel="stylesheet"/>
 
 <button routerLink="/"> Home </button>
-<button *ngIf="!isAuthenticated" routerLink="/login"> Login </button>
-<button *ngIf="isAuthenticated" (click)="logout()"> Logout </button>
+<button *ngIf="!(authStateService.authState$ | async).isAuthenticated" routerLink="/login"> Login </button>
```

Comment:
this seems like a downgrade in developer experience. It is much better if the UI is not directly binding to async actions, but the entire UI can be rendered synchronously.

Reply:
I actually think the `async pipe` is the suggested approach for observable when a default value is provided. In this way, devs do not need to handle `subscribe` and `unsubscribe` of each observable values, which may cause memory leak if `unsubscribe` is missed from their components.